### PR TITLE
feat(server-cache): estimatedSize

### PR DESCRIPTION
**Pull request by JAdshead**

### CODE REVIEW

**Review by nellyk — DISMISSED** [review]

**Comment by nellyk** [comment]
Can you look and check why the tests are failing https://pipelines.actions.githubusercontent.com/3svxpBQt1h6LKX5TG2I0iS0kPH77L8frPgypt3Bq5aDaNqz9vI/_apis/pipelines/1/runs/495/signedlogcontent/3?urlExpires=2021-01-05T14%3A16%3A41.8049940Z&urlSigningMethod=HMACV1&urlSignature=GSluvBrerwSdj%2FwPBHu6QSTwrPakexpfUWbRSsmj1jU%3D

File: README.md [hunk 1]
```diff
@@ -601,6 +601,18 @@ import { setDangerouslyDisableScriptsAndStyles } from '@americanexpress/one-app-
 dispatch(setDangerouslyDisableScriptsAndStyles(true));
 ```
 
+#### Intl Cache
+
+##### estimatedServerCacheSize
+
+Returns the estimated size of the Intl server cache in Bytes. This is accurate enough to track changes to the cache size overtime however, as there is no native `sizeOf` function in Javascript (like in C), the actual memory allocation of the cache will differ.
```
Comment (10xLaCroixDrinker) [inline README.md:608]:
```suggestion
Returns the estimated size of the Intl server cache in Bytes. This is accurate enough to track changes to the cache size over time however, as there is no native `sizeOf` function in Javascript (like in C), the actual memory allocation of the cache will differ.
```

File: src/index.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ import intlReducer, {
   queryLanguagePack,
   updateLocale,
 } from './intl';
+import { estimatedSize as estimatedServerCacheSize } from './intl/server-cache';
```
Comment (10xLaCroixDrinker) [inline src/index.js:24]: Can we keep this out of the client API & bundle?

File: src/intl/server-cache.js [hunk 3]
```diff
@@ -35,9 +37,12 @@ function get(url) {
   return cached.data;
 }
 
+const estimatedSize = () => sizeOf(Object.fromEntries(cache));
+
 export {
   set,
   get,
+  estimatedSize,
```
Comment (10xLaCroixDrinker) [inline src/intl/server-cache.js:45]:
```suggestion
const getEstimatedSize = () => sizeOf(Object.fromEntries(cache));

export {
  set,
  get,
  getEstimatedSize,
```

File: README.md [hunk 4]
```diff
@@ -601,6 +601,18 @@ import { setDangerouslyDisableScriptsAndStyles } from '@americanexpress/one-app-
...
+
+```js
+import { estimatedServerCacheSize } from '@americanexpress/one-app-ducks';
```
Comment (10xLaCroixDrinker) [inline README.md:611]:
```suggestion
import { getEstimatedServerCacheSize } from '@americanexpress/one-app-ducks';
```

File: package.json [hunk 5]
```diff
@@ -54,10 +54,11 @@
     ]
   },
   "dependencies": {
+    "holocron": "^1.0.0-9",
```
Comment (10xLaCroixDrinker) [inline package.json:57]: Take this opportunity to get this off a prerelease

**Review by 10xLaCroixDrinker — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (JAdshead) [reply src/index.js:24]: sure

File: README.md (hunk 4 above)
Comment (JAdshead) [reply README.md:611]: will remove from README as dropping from public api

**Review by infoxicator — APPROVED** [review]

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by nellyk — APPROVED** [review]

**Pull request merged**